core/src/xtdb/tx.clj

```diff
@@ -378,19 +376,17 @@
 
 (defrecord TxIndexer [index-store document-store bus query-engine]
   db/TxIndexer
-  (begin-tx [_ tx fork-at]
-    (when-not fork-at
-      (log/debug "Indexing tx-id:" (::xt/tx-id tx))
-      (bus/send bus {::xt/event-type ::indexing-tx, :submitted-tx tx}))
+  (begin-tx [_ tx]
+
+    (log/debug "Indexing tx-id:" (::xt/tx-id tx))
+    (bus/send bus {::xt/event-type ::indexing-tx, :submitted-tx tx})
 
-    (let [index-store-tx (db/begin-index-tx index-store tx fork-at)
+    (let [index-store-tx (db/begin-index-tx index-store tx)
           document-store-tx (fork/begin-document-store-tx document-store)]
-      (->InFlightTx tx fork-at
+      (->InFlightTx tx
                     (atom :open)
-                    (atom {:doc-ids #{}
-                           :av-count 0
-                           :bytes-indexed 0
-                           :tx-events []})
+                    (atom {:tx-events [] :av-count 0 :bytes-indexed 0 :doc-ids #{}})
```

Comment:
subjective code style: I don't mind this being on one line, but if so I tend to put commas between the kvs to visually delineate them for the reader: `{:tx-events [], :av-count 0, ...}`